components/function-runtimes/python38/kubeless/kubeless.py

```diff
@@ -102,26 +112,12 @@ def handler():
         'ce-time': req.get_header('ce-time'),
         'extensions': {'request': picklable_req}
     }
+
     method = req.method
     func_calls.labels(method).inc()
     with func_errors.labels(method).count_exceptions():
         with func_hist.labels(method).time():
-            q = ctx.Queue()
-            p = ctx.Process(target=funcWrap, args=(q, event, function_context))
-            p.start()
-
-            try:
-                res = q.get(block=True, timeout=timeout)
-            except queue.Empty:
-                p.terminate()
-                p.join()
-                return bottle.HTTPError(408, "Timeout while processing the function")
-            else:
-                p.join()
-                if isinstance(res, Exception) and not isinstance(res, bottle.HTTPResponse):
-                    logging.error("Function returned an exception: %s", res)
-                    raise res
-                return res
+            return func(event, function_context)
```

Comment:
The way you changed this means we do not have timeout errors anymore, am I correct?

Reply:
I can add the timeout mechanism.

Reply:
@rJankowski93 and I managed the timeout issue